lib/tether/tether.go

```diff
@@ -251,8 +251,18 @@ func (t *tether) setHostname() error {
 }
 
 func (t *tether) setNetworks() error {
+	var networks []*NetworkEndpoint
+	// prioritize internal networks
 	for _, v := range t.config.Networks {
-		if err := t.ops.Apply(v); err != nil {
+		if !v.Internal {
```

Comment:
Very cosmetic comment: It doesn't really change anything, especially for the binary output, however I would remove "!" and swap if/else clauses.